tests/python/22_verify/22_verify.py

```diff
@@ -91,68 +57,55 @@ def runKernel(opt):
     # Prepare kernel reg map
     new_data = (ctypes.c_uint32 * rsz).from_buffer(execData.contents, 8)
     new_data[XHELLO_HELLO_CONTROL_ADDR_AP_CTRL] = 0x0
-    new_data[int(XHELLO_HELLO_CONTROL_ADDR_ACCESS1_DATA / 4)] = bodevAddr
-    new_data[int(XHELLO_HELLO_CONTROL_ADDR_ACCESS1_DATA / 4 + 1)] = (bodevAddr >> 32) & 0xFFFFFFFF
-
-    if xclExecBuf(opt.handle, execHandle):
-        xclFreeBO(opt.handle, boHandle)
-        xclFreeBO(opt.handle, execHandle)
-        xclCloseContext(opt.handle, opt.xuuid, 0)
-        print("Error: Unable to issue exec buf")
-        return 1
-    else:
-        print("Kernel start command issued through xclExecBuf : start_kernel")
-        print("Now wait until the kernel finish")
+    new_data[int(XHELLO_HELLO_CONTROL_ADDR_ACCESS1_DATA / 4)] = p.paddr
+    new_data[int(XHELLO_HELLO_CONTROL_ADDR_ACCESS1_DATA / 4 + 1)] = (p.paddr >> 32) & 0xFFFFFFFF
+
+    xclExecBuf(opt.handle, execHandle)
+    print("Kernel start command issued through xclExecBuf : start_kernel")
+    print("Now wait until the kernel finish")
```

Comment:
So, is the idea with this example to demo the xclExec* native APIs as opposed to the kernel APIs?